steps/firebase-change-file-by-env/0.0.1/step.yml

```diff
@@ -0,0 +1,83 @@
+title: Firebase Change File by environment
+summary: |
+  Change file change firebase file according to branch
+description: |
+  Change file between firebase project according to branch in your repository
+  * Important Suggestion *: You must create a folder in the root of your project called 'firebase_env'
+   and inside have two folders for production and development, the step suggests in its variables
+   and the corresponding folders.
```

Comment:
Those texts overall seem to be insufficient.
It is hard to imagine what this step do.
Summary is too laconic.

Try to rephrase and fix errors like `Change file change`